src/features/common/data/atoms-in-atom.ts

```diff
@@ -1,12 +1,11 @@
 import { WritableAtom, atom, type Atom } from 'jotai'
 import { dataStore } from './data-store'
 
-const defaultCreateInitialValueAtom = <T>(value: T) => atom(() => value)
-
 export const createTimestamp = (): number => Date.now()
 
 export const createAtomAndTimestamp = <T>(value: T) => {
-  return [defaultCreateInitialValueAtom(value), createTimestamp()] as const
+  const createAtomFunc = (value: T) => atom(() => Promise.resolve(value))
```

Comment:
Probably no need to have this defined as a function anymore. It can be inlined. It's also interesting that we need to do a Promise.resolve now. Is that needed?